geomstats/geometry/discrete_curves.py

```diff
@@ -1270,3 +1273,39 @@ def horizontal_path(t):
             return horizontal_path
 
         return horizontal_path
+
+    def quotient_dist(self, point_a, point_b, n_times=20, threshold=1e-3):
```

Comment:
For clarity for non-experts, maybe specify that this the quotient with respect to "parameterizations", i.e. with respect to sampling along the curve?